src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -252,6 +256,19 @@ internal static void FreeSslContext(IntPtr sslContextPtr)
 
         #region private methods
 
+        private static void FreeBio(SslContext sslContext)
+        {
+            if (IntPtr.Zero != sslContext.readBioPtr)
+            {
+                Interop.libcrypto.BIO_free(sslContext.readBioPtr);
+            }
+
+            if (IntPtr.Zero != sslContext.writeBioPtr)
+            {
+                Interop.libcrypto.BIO_free(sslContext.writeBioPtr);
+            }
```

Comment:
When SslContext is changed to be a class, this should null out the readBioPtr and writeBioPtr fields as well.